src/current/molt/molt-fetch.md

```diff
@@ -68,6 +68,8 @@ Complete the following items before using MOLT Fetch:
 	- **Maximum allowed number of connections:** MOLT Fetch can export data across multiple connections. The number of connections it will create is the number of shards ([`--export-concurrency`](#global-flags)) multiplied by the number of tables ([`--table-concurrency`](#global-flags)) being exported concurrently.
 	- **Maximum lifetime of a connection:** This is particularly important for MySQL sources, which can only use a single connection to move data. See the following note.
 
+- If a PostgreSQL database is set as a [source](#source-and-target-databases), ensure that [`idle_in_transaction_session_timeout`](https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-IDLE-IN-TRANSACTION-SESSION-TIMEOUT) on PostgreSQL is either disabled or set to a value longer than the duration of the Fetch task. Otherwise, the connection will be prematurely terminated. To estimate the Fetch duration, you can [perform a dry run](#perform-a-dry-run) and observe the `molt_fetch_overall_duration` metric.
```

Comment:
Sorry I should have made the comment more clear in the ticket. "longer than the duration of the Fetch task" should apply only to the data export phase since after than we no longer need the idle transaction.

Reply:
Got it, also changed the metric to `molt_fetch_table_export_duration_ms`.